src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1161,14 +1430,38 @@ void convertVectorFromDegreesToRadians(iDynTree::VectorDynSize & vector)
 bool WholeBodyDynamicsDevice::readFTSensors(bool verbose)
 {
     bool FTSensorsReadCorrectly = true;
+    bool TempSensorReadCorrectly = true;    
+    double timeFTStamp=yarp::os::Time::now();
+    bool readTemperatureSensorThisTime=false;
+    yarp::dev::MAS_status    sensorStatus;
     for(size_t ft=0; ft < estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE); ft++ )
     {
         iDynTree::Wrench bufWrench;
         int ftRetVal = ftSensors[ft]->read(ftMeasurement);
+        if (timeFTStamp-prevFTTempTimeStamp>0.55){
```

Comment:
Why 0.55 ? Can we make this a parameter loadable from configuration file with 0.55 as a default value?